htlcswitch/link.go

```diff
@@ -659,7 +658,7 @@ func (l *channelLink) handleDownStreamPkt(pkt *htlcPacket, isReProcess bool) {
 	case *lnwire.UpdateFailHTLC:
 		// An HTLC cancellation has been triggered somewhere upstream,
 		// we'll remove then HTLC from our local state machine.
-		logIndex, err := l.channel.FailHTLC(pkt.payHash)
+		err := l.channel.FailHTLC(pkt.destID)
```

Comment:
I find the src/dest breakdown a bit confusing here. In my mind, this should be the src ID as a previously _outgoing_ HTLC (from the PoV of this channel) has now been cancelled by the dest link (where we extended the forwarded HTLC). 

What do you think of switching to incoming/outgoing?

Reply:
Yeah, I agree the incoming/outgoing terminology is less ambiguous. Will change.